src/Redux.js

```diff
@@ -5,7 +5,7 @@ export default class Redux {
   constructor(dispatcher, initialState) {
     if (typeof dispatcher === 'object') {
       // A shortcut notation to use the default dispatcher
-      dispatcher = createDispatcher(composeStores(dispatcher));
+      dispatcher = createDispatcher(composeStores(dispatcher), initialState);
```

Comment:
@emmenko It's because we're overloading the signature of the constructor. `initialState` is actually `middleware`, just like `dispatcher` in that call is actually a stores hash. You make a good point, though... it should be the third argument so we can still set the `initialState`. Eventually we'll need to rethink this (perhaps by using an options object instead) but for now we're trying not to break the existing API.